tsconfig.json

```diff
@@ -1,5 +1,8 @@
 {
   "compilerOptions": {
+    "target": "es2018",
+    "module": "node16",
+    "moduleResolution": "node16",
```

Comment:
Ensure the typescript type-checking [supports the ES module](https://www.typescriptlang.org/docs/handbook/esm-node.html)